### DISCUSSION TITLE
Marker

### DISCUSSION THREAD

**Discussion by EMIN3M1986** [patrykandpatrick/vico#507] [Questions]
How can I have marker like the video. (or something like this

https://github.com/patrykandpatrick/vico/assets/5969144/527899fc-bcd9-4e3c-abf6-6ab664d839ca

)

**Comment by Gowsky** [comment]
Hi @EMIN3M1986, unfortunately, this is not possible. You may be able to achieve it by extending the API by yourself.

**Reply by patrickmichalik** [reply]
Hello! To expand on this, `Marker`s are drawn above everything else, and while they aren’t limited to the `CartesianLayer` area (or, in Vico 1, the `Chart` area), they aren’t given information on their charts’ axes. Since `HorizontalAxis` instances often have static dimensions (dependent on such parameters as the label font size), the latter limitation isn’t necessarily a deal-breaker. The former limitation, however, makes it impossible for a `Marker` to display opaque content under its chart. Thus, as mentioned above by @Gowsky, a custom solution would be required here.
